Queueomatic/Client/Components/SignUp/SignUpComponent.razor

```diff
@@ -0,0 +1,58 @@
+﻿@using Blazored.SessionStorage
+@inject HttpClient HttpClient;
+@inject NavigationManager NavigationManager;
+@inject ISessionStorageService SessionStorageService;
+
+<LogoComponent/>
+<AuthorizeView>
+    <Authorized>
+        <RedirectHome/>
+    </Authorized>
+</AuthorizeView>
+<div class="component">
+    <div class="form-container">
+        <p class="title">Signup</p>
+        <EditForm class="form" Model="_signUpDto" OnValidSubmit="SignUp">
+            <DataAnnotationsValidator/>
+            <div class="input-group">
+                <label for="username">Nickname</label>
+                <input type="text" name="username" placeholder="" class="input-field" id="username" @bind-value="_signUpDto.NickName">
+                <ValidationMessage style="color: #2e2e2e !important; font-size: 13px; padding-top: 4px;" For="(() => _signUpDto.NickName )"/>
+                @* <div class="invalid-feedback"> *@
```

Comment:
What is the importance of these comments?

Reply:
None. Forgot to remove them.